floodviz/thumbnail/thumbnail.js

```diff
@@ -15,6 +15,7 @@ var jsdom = require('jsdom/lib/old-api.js');
 var svg2png = require('svg2png');
 // Data imports
 var data_hydro = require('../thumbnail/hydrograph_data.json');
+var reference = require('../../examples/reference.json');
```

Comment:
This location needs instance/reference.json as that is where we are putting it in the dockerfile process.